oaknational/Oak-Web-Application
# Feat/lesq 885/combined units page for maths [LESQ-885]

## Description

Music year: 1988

- update components library
- use new unit container component on unit listing page for maths only
- update fixtures and tests

## How to test

1. Go to {owa_deployment_url}
2. Navigate to unit listing pages for maths across key stages
3. You should see the new layout
4. You should not see the new layout for other subjects

## Screenshots

How it should now look:
<img width="600" alt="Screenshot 2024-06-20 at 10 10 00" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/43c58725-e649-4d07-87e1-bedc1b14348f">


<img width="600" alt="Screenshot 2024-06-20 at 08 50 03" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/0806dc0b-ce56-4408-9374-a6e1b0b39504">

## Checklist

- [ ] Header for units new and legacy should only appear once
- [ ] Numbering for legacy units should start from 1
- [ ] numbering should continue correctly onto subsequent pages (calling out as this was fiddly)
- [ ] Curriculum download buttons should go to the correct place for new/legacy
- [ ] Thread filters should work as before (only for new content)
- [ ] There should be 20 units per page as before
